tools/dm/table-selector.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Table Selector
+summary: 介绍 DM 的 Table Selector
+category: tools
+---
+
+# Table Selector
+
+Table Selector 提供了一种基于 [通配符](https://zh.wikipedia.org/wiki/%E9%80%9A%E9%85%8D%E7%AC%A6) 来匹配指定 `schema/table` 的功能。
+
+## 通配符
+
+table selector 在 `schema-pattern`/`table-pattern` 中可以使用以下两个通配符：
+
++ 星号(`*`)
+
+  - 匹配零个或者多个字符。例如， `doc*` 匹配 `doc` 和 `document`，但是不匹配 `dodo`。
+  - `*` 只能放在 pattern 的最后一位，例如，支持 `doc*`, 但是不支持 `do*c`。
+
++ 问号(`?`)
+
+  - 匹配任意一个空字符除外的字符。
```

Comment:
is `?` matches zero or one occurrences of the preceding element ?

Reply:
one